packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -8099,6 +8099,333 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /eservices/{eserviceId}/descriptors/{descriptorId}/certifiedAttributes/groups:
+    parameters:
+      - name: eserviceId
+        in: path
+        description: The E-Service ID of the Descriptor
+        required: true
+        schema:
+          type: string
+          format: uuid
+      - name: descriptorId
+        in: path
+        description: The ID of the Descriptor
+        required: true
+        schema:
+          type: string
+          format: uuid
+    post:
+      tags:
+        - eservices
+      summary: Create Certified Attributes Group for E-Service Descriptor
```

Super minor align with other create calls (same for the other new calls in the PR)
```suggestion
      summary: Create a Certified Attributes Group for an E-Service Descriptor
```